pkg/controller/restore/restore_controller.go

```diff
@@ -148,8 +150,34 @@ func (c *Controller) updateRestore(cur interface{}) {
 		return
 	}
 
-	if v1alpha1.IsRestoreScheduled(newRestore) {
-		klog.V(4).Infof("restore %s/%s is already scheduled, skipping", ns, name)
+	if v1alpha1.IsRestoreFailed(newRestore) {
```

Comment:
Please add some UT cases to cover the code change in this file and the backup_controller.go.

Reply:
added in https://github.com/pingcap/tidb-operator/pull/3696/commits/3aaa29eb0559e8082b59601ad54d8ebeaf5b0931.